### PR TITLE
support for image data drift alerts

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: dkube/sdk/rsrcs/modelmonitor.py [hunk 1]
```diff
@@ -701,7 +701,7 @@ def __init__(self, name="mm-alert", tags=[]):
     def to_JSON(self):
         return json.dumps(self, default=lambda o: o.__dict__)
 
-    def update_alert(
+    def add_alert_condition(
```
Comment (songole) [inline dkube/sdk/rsrcs/modelmonitor.py:704]: I thought you are retaining the existing update_alert so QA doesn't have to change scripts? @SireeshaMandava

Comment (riteshkarval) [reply dkube/sdk/rsrcs/modelmonitor.py:704]: yes @songole

File: dkube/sdk/rsrcs/modelmonitor.py [hunk 2]
```diff
@@ -753,9 +751,9 @@ def update_alert(
...
             }
         )
-       
         if breach_threshold:
             self.alert_action["breach_threshold"] = breach_threshold
         if emails:
             self.alert_action["emails"] = emails
-            
+
+    update_alert = add_alert_condition
```
Comment (riteshkarval) [inline dkube/sdk/rsrcs/modelmonitor.py:759]: @songole here I am keeping update_alert function, so that QA don't need to change immediately.

Comment (riteshkarval) [reply dkube/sdk/rsrcs/modelmonitor.py:759]: @SireeshaMandava

**Pull request merged**